app/src/pages/inside/productVersionPage/linkedTestCasesTab/headerConfig/headerConfig.jsx

```diff
@@ -44,6 +44,12 @@ export const HeaderConfig = ({ filters, setFilters }) => {
     setIsMainFilterOpen(false);
     setIsInnerFilterOpen(false);
   };
+  // Implemented only for fixing sonar cube errors
+  const handleKeyDown = (e) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      console.log(e);
```

Comment:
Shouldn't this handler perform the same action as click listener?

Reply:
Let's also use `keyCodes` defined [here](https://github.com/reportportal/service-ui/blob/develop/app/src/common/constants/keyCodes.js) for consistency and readability.